React/rematch/src/models.js

```diff
@@ -0,0 +1,20 @@
+export const items = {
+  state: {
+    newItem: "",
+    allItems: ["Hot Dog", "Pizza", "Donuts"]
+  },
+  reducers: {
+    setNewItemText: (state, payload) => ({
+      ...state,
+      newItem: payload
+    }),
+    addItem: state => ({
+      ...state,
+      allItems: state.allItems.concat([state.newItem])
```

Comment:
### you need to clear out newItem here.  Otherwise, the text box keeps old state on adding and it ends up like this:


![image](https://user-images.githubusercontent.com/997157/40242829-48645ce2-5a84-11e8-856f-e084dec67405.png)

Reply:
Yep :dagger: